### DISCUSSION TITLE
Reverse connection

### DISCUSSION THREAD

**Discussion by dsanttos12** [ClassicOldSong/Apollo#74] [Ideas]
It would be a good idea to implement reverse connection on Artemis and Apollo. Through the reverse connection with my computer with the TCP and UDP ports redirected and with the public IP fixed, I will be able to access the machines of friends whose computers are blocked by NAT. It is very laborious for them to redirect TCP and UDP ports. They need to contact the operator's support to request port redirection. Reverse connection is a simple feature that would be easy to add.

**Comment by ClassicOldSong** [comment]
If you can reverse connect, just use Tailscale or ZeroTier or some other similar P2P VPN solutions. They have very little impact on latency.
